go/service/wot.go

```diff
@@ -88,10 +88,7 @@ func (h *WebOfTrustHandler) WotVouchCLI(ctx context.Context, arg keybase1.WotVou
 func (h *WebOfTrustHandler) WotListCLI(ctx context.Context, arg keybase1.WotListCLIArg) (res []keybase1.WotVouch, err error) {
 	ctx = libkb.WithLogTag(ctx, "WOT")
 	mctx := libkb.NewMetaContext(ctx, h.G())
-	if arg.Username == nil {
-		return libkb.FetchMyWot(mctx)
-	}
-	return libkb.FetchUserWot(mctx, *arg.Username)
+	return libkb.FetchUserWotByVoucher(mctx, *arg.Vouchee, *arg.Voucher)
```

Comment:
are vouchee and voucher _always_ non-nil here? you cannot dereference a nil pointer without having a bad time.